security_monkey/watchers/openstack/openstack_watcher.py

```diff
@@ -0,0 +1,127 @@
+#     Copyright (c) 2017 AT&T Intellectual Property. All rights reserved.
+#
+#     Licensed under the Apache License, Version 2.0 (the "License");
+#     you may not use this file except in compliance with the License.
+#     You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+"""
+.. module: security_monkey.openstack.watchers.openstack_watcher
+    :platform: Unix
+
+.. version:: $$VERSION$$
+.. moduleauthor:: Michael Stair <mstair@att.com>
+
+"""
+
+from security_monkey.cloudaux_watcher import CloudAuxWatcher, CloudAuxChangeItem
+from security_monkey.decorators import record_exception
+
+from cloudaux.openstack.decorators import iter_account_region, get_regions
+from cloudaux.openstack.utils import list_items
+from cloudaux.orchestration.openstack.utils import get_item
+
+
+
+class OpenStackWatcher(CloudAuxWatcher):
+    def __init__(self, accounts=None, debug=False):
+        super(OpenStackWatcher, self).__init__(accounts=accounts, debug=debug)
+        self.honor_ephemerals = True
+        self.ephemeral_paths = [ "updated_at" ]
+
+    def _get_openstack_creds(self, account):
+        from security_monkey.datastore import Account
+
+        _account = Account.query.filter(Account.name.in_([account])).one()
+        return _account.identifier, _account.getCustom('cloudsyaml_file')
+
+    """ Regions are not global but account specific """
```

Comment:
I thought docstrings went after the method signature.  Does this work?  Is it still called a docstring?

Reply:
No, got comment happy, I will fix.